Fix mistake on creating certificates when connecting app to VNet

radu-matei opened the pull request:
You don't need to create certificates manually when connecting an App Service app to a private network through Point-to-Site connection, that is done automatically. You only need to specify the client address space.

azurecla commented:
Hi __@DanSeefeldt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

Commit:
Fix mistake on creating certificates

You don't need to create certificates manually when connecting an App Service app to a private network through Point-to-Site connection, that is done automatically. You only need to specify the client address space.
